.dockerignore

```diff
@@ -0,0 +1,15 @@
+# Ignore files from the docker context
+**/__pycache__
+**/tests
+**/.coverage
+**/.git
+.gitignore
+.pytest_cache
+.vagrant
+.vscode
+Dockerfile
+Vagrantfile
+chromedriver.exe
+poetry.lock
+*.log
+*.env*
```

Comment:
It would be good to also add the `.venv` folder here to prevent a developers virtual environment getting copied in